packages/ui-extensions/docs/surfaces/point-of-sale/reference/apis/cart-api.doc.ts

```diff
@@ -1,10 +1,16 @@
 import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
-import {generateCodeBlock} from '../helpers/generateCodeBlock';
+import {
+  generateCodeBlock,
+  generateJsxCodeBlock,
+} from '../helpers/generateCodeBlock';
 import {ExtensionTargetType, TargetLink} from '../types/ExtensionTargetType';
 
 const generateCodeBlockForCartApi = (title: string, fileName: string) =>
```

Comment:
Still need this as it generates the code for all the rest of the examples. I'll follow up to completely remove this